src/re_graph/internals.cljc

```diff
@@ -91,7 +91,7 @@
    2. Response is a map but does not contain a valid errors map: merge in default errors
    3. Response is anything else: return default errors map"
   [response status]
-  (let [f (fn [errors] (mapv #(assoc-in % [:extensions :status] status) errors))
+  (let [f (fn [errors] (mapv (fn [error] (update-in error [:extensions :status] #(or % status))) errors))
```

Comment:
The errors may have many different status values. By always overwriting the status, information is lost. This change will only insert the HTTP status (in a valid graphql response) if there isn't already a status value.